### ISSUE TITLE
auto-complete activated by period in f-string

### ISSUE THREAD

**Issue by hudcap** [microsoft/pylance-release#1226]
<h3> Environment data </h3>

-   Language Server version: 2021.4.3
-   OS and version: win32 x64
-   Python version (and distribution if applicable, e.g. Anaconda): 3.9.5 64-bit
-   python.analysis.indexing: undefined
-   python.analysis.typeCheckingMode: basic

<h3> Expected behaviour </h3>

In an f-string, auto-complete should only activate inside braces.

<h3> Actual behaviour </h3>

Auto-complete activates when typing a period inside an f-string.

![image](https://user-images.githubusercontent.com/4990645/117029035-46704200-accc-11eb-9b4b-0840996df4e3.png)

Unless there's some significance to a period in an f-string that I'm not aware of?

<h3> Logs </h3>

Python Language Server Log
```
Background analysis message: setFileOpened
Background analysis message: markFilesDirty
Background analysis message: setFileOpened
Background analysis message: markFilesDirty
Background analysis message: getSemanticTokens delta
[BG(1)] getSemanticTokens delta previousResultId:1620142513034 at Untitled-1 ...
[BG(1)]   parsing: Untitled-1 (0ms)
[BG(1)]   binding: Untitled-1 (0ms)
[BG(1)] getSemanticTokens delta previousResultId:1620142513034 at Untitled-1 (0ms)
Background analysis message: setFileOpened
Background analysis message: markFilesDirty
Background analysis message: getDiagnosticsForRange
Background analysis message: getDiagnosticsForRange
[FG] parsing: Untitled-1 (0ms)
[FG] binding: Untitled-1 (0ms)
Background analysis message: getSemanticTokens delta
[BG(1)] getSemanticTokens delta previousResultId:1620142522108 at Untitled-1 ...
[BG(1)]   parsing: Untitled-1 (0ms)
[BG(1)]   binding: Untitled-1 (1ms)
[BG(1)] getSemanticTokens delta previousResultId:1620142522108 at Untitled-1 (1ms)
Background analysis message: analyze
[BG(1)] analyzing: Untitled-1 ...
[BG(1)]   checking: Untitled-1 (2ms)
[BG(1)] analyzing: Untitled-1 (2ms)
Background analysis message: resumeAnalysis
Background analysis message: setFileOpened
Background analysis message: markFilesDirty
[FG] completion at Untitled-1:0:12 ...
[FG]   parsing: Untitled-1 (2ms)
[FG]   binding: Untitled-1 (0ms)
[FG] completion at Untitled-1:0:12 [found 189 items] (5ms)
Background analysis message: analyze
[BG(1)] analyzing: Untitled-1 ...
[BG(1)]   parsing: Untitled-1 (0ms)
[BG(1)]   binding: Untitled-1 (0ms)
[BG(1)]   checking: Untitled-1 (2ms)
[BG(1)] analyzing: Untitled-1 (3ms)
Background analysis message: getDiagnosticsForRange
Background analysis message: resumeAnalysis
Background analysis message: getDiagnosticsForRange
Background analysis message: getSemanticTokens delta
[BG(1)] getSemanticTokens delta previousResultId:1620142522554 at Untitled-1 (0ms)
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in an upcoming release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.5.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202150-5-may-2021
